app/src/main/java/com/xa/xpensauditor/Day_wise_visualization.java

```diff
@@ -137,33 +141,54 @@ public void onDataChange(@NonNull DataSnapshot datasnapshot) {
                             System.out.println(" Finallyyy   "+amm);
                             int am1=Integer.valueOf(amm);
                             tot=am1+tot;
-      dumm=snapshot.child("Day").getValue().toString();
+                            dumm=snapshot.child("Day").getValue().toString();
+                            dayyy1=Integer.valueOf(dumm);
+                            Calendar c1 = Calendar.getInstance();
+                            c1.set(y2, m2-1, dayyy1, 0, 0);
+                            Date date = new GregorianCalendar(y2, m2-1, dayyy1).getTime();
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '-' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)